src/components/scenes/HomeScene.tsx

```diff
@@ -95,7 +95,7 @@ export const HomeScene = (props: Props) => {
   // Set countryCode once
   useAsyncEffect(
     async () => {
-      const countryCode = await getCountryCodeByIp().catch(() => '')
+      const countryCode = await getCountryCodeByIp().catch(() => undefined)
```

Comment:
`getCountryCodeByIp` also has to be fixed as it returns `--` if it can't get a countryCode. This was my oversight.